charts/core/values/postgresql.yaml

```diff
@@ -6,11 +11,32 @@ primary:
 
         set -euo pipefail
 
-        ABSOLUTE_SEED_URL="https://databases.dhis2.org/sierra-leone/2.36.0/dhis2-db-sierra-leone.sql.gz"
-        curl -L "$ABSOLUTE_SEED_URL" -o /tmp/t$$ | cat
-        gunzip -c /tmp/t$$ > /tmp/t$$-seed-data
-        psql -U postgres -d dhis2 -p 5432 -f /tmp/t$$-seed-data
-        rm /tmp/t$$ /tmp/t$$-seed-data
+        export PGPASSWORD=$POSTGRES_POSTGRES_PASSWORD
+
+        function exec_psql() {
+          psql -U postgres -qAt -d dhis2 -c "$1"
+        }
+
+        table_exists=$(exec_psql "select exists (select from information_schema.tables where table_schema = 'schema_name' and table_name = 'table_name')")
+        if [[ $table_exists = true ]]; then
+          row_count=$(exec_psql "select count(*) from organisationunit")
```

Comment:
Not sure how many orgunits are in a DB, but in general I would try to use something like `select organisationunitid from organisationunit limit 1;` instead of a `count(*)` to not even have to worry about that 😅